extensions/sql-migration/src/constants/helper.ts

```diff
@@ -110,6 +110,22 @@ const _numberFormatter = new Intl.NumberFormat(
 	maximumFractionDigits: 0,
 });
 
+export function formatSecondsIntoReadableTime(seconds: number) {
+	const hours = seconds / (60 * 60);
+	const absoluteHours = Math.floor(hours);
+	const h = absoluteHours > 9 ? absoluteHours : '0' + absoluteHours;
+
+	const minutesRemaining = (hours - absoluteHours) * 60;
+	const absoluteMinutes = Math.floor(minutesRemaining);
+	const m = absoluteMinutes > 9 ? absoluteMinutes : '0' + absoluteMinutes;
+
+	const secondsRemaining = (minutesRemaining - absoluteMinutes) * 60;
+	const absoluteSeconds = Math.floor(secondsRemaining);
+	const s = absoluteSeconds > 9 ? absoluteSeconds : '0' + absoluteSeconds;
```

Comment:
instead of appending strings, can a number formatter be used instead to avoid local conversion issues.

const _numberFormatter = new Intl.NumberFormat(
	undefined, {
	style: 'decimal',
	useGrouping: false,
	minimumIntegerDigits: 2,
	minimumFractionDigits: 0,
	maximumFractionDigits: 0,
});